contracts/VestedPayment.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity ^0.4.13;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./LifToken.sol";
+
+contract VestedPayment is Ownable {
+  using SafeMath for uint256;
+
+  // when the vested schedule starts
+  uint256 public startTime;
+
+  // how much each period will last
+  uint256 public timePerPeriod;
```

Comment:
what's the unit of this? I think it's seconds, it should be part of the name, and also in the comment

Reply:
:+1: